README.md

```diff
@@ -1,3 +1,32 @@
 # RHMAP Nagios for Docker
 
 Nagios server in a Docker container for RHMAP.
+
+# Setting up nagios service account
+
+In order to talk to the OpenShift and perform exec commands on pods, a service account will need to be created for use by nagios and given the proper permissions.
+
+1. Create the account using oc in the same project as the mbaas/core you are monitoring:
+
+```
```

Comment:
You need to indent this, otherwise you get two independent numbered lists starting from 1, see:

https://github.com/jessesarn/nagios-container/blob/5f915f9b8bf8fa9efd4597f9e55ad50a6db010c1/README.md